docker-compose.override.no-persist.yml

```diff
@@ -0,0 +1,5 @@
+volumes:
+  esdata:
+  s3data:
+  redisdata:
+  amqpdata:
```

Comment:
When this file is renamed to `docker-compose.override.yml` the default behavior is to override the `docker-compose.yml` - thereby not persisting data locally (if desired)